superset/connectors/sqla/views.py

```diff
@@ -57,8 +57,8 @@ class TableColumnInlineView(CompactCRUDMixin, SupersetModelView):  # noqa
             'expression-defined columns in some cases. In most case '
             'users should not need to alter this.'),
         'expression': utils.markdown(
-            'a valid SQL expression as supported by the underlying backend. '
-            'Example: `substr(name, 1, 1)`', True),
+            'a valid, non-aggregating SQL expression as supported by the '
```

Comment:
Nit: use stars around "non-aggregating", for consistency with the other chance below.